ethereum-contracts/README.md

```diff
@@ -1,72 +1,61 @@
-# Testing contracts
+# Unidirectional Peggy Project Specification
 
-## Installing dependencies
+## Project Summary
+Unidirectional Peggy is the starting point for cross chain value transfers from Ethereum to Cosmos as part of the Ethereum Cosmos Bridge project. The smart contract system accepts incoming transfers of Ethereum and ERC20 tokens, locking them while the transaction is validated and equitable funds issued to the intended recipient on Cosmos.
 
-  - Truffle: `npm install -g truffle` for Mac or go to the install
-  - Testrpc: `npm i -g ethereumjs-testrpc`
-  - Ganache: Download file [here](http://truffleframework.com/ganache/)
+## Project Background
+We are hoping to create a closed system for intra network transfers of cryptocurrency between blockchains, spearheaded by a proof-of-concept which enables secured transactions between Ethereum and Cosmos.
 
-## Instructions
+## Smart Contract Scope
 
-You can test the contracts using Truffle Testrpc in the console or Ganache UI.
+### Goals of the Smart Contracts
+1. Securely implement core functionality of the system such as asset locking and event emission without endangering any user funds. As such, this prototype does not permanently lock value and allows the original sender full access to their funds at any time.
```

Comment:
What do you mean by "permanently lock"? Can the original sender unlock their original tokens even after they've sent tokens to Cosmos?

Reply:
Correct. The original sender can withdraw tokens using `withdraw(bytes32 id)` while the contract's relayer can return tokens to the item's original sender using `unlock(id)`. This is intended behavior for this stage of the project to protect any user funds should someone attempt to deploy the contracts to the mainnet; once the bridge has undergone robust testing the `withdraw()` function will be removed entirely and the `unlock()` function will be contingent on approval from the validator set.

I've updated the project specification to state that this project specification focuses on the role of 'Peggy', the Smart Contract system deployed to the Ethereum network, and is meant to contextualize its role within the bridge. Specifications detailing the structure and step=-by-step process of the non-Ethereum components (Relayer service, EthOracle module, Oracle module) will soon be available in the cosmos-ethereum-bridge repository.